laser_encoders/download_models.py

```diff
@@ -46,21 +46,31 @@ def __init__(self, model_dir: str = None):
 
     def download(self, filename: str):
         url = os.path.join(self.base_url, filename)
-        local_file_path = self.model_dir / filename
 
-        if local_file_path.exists():
+        local_file_path = os.path.join(self.model_dir, filename)
+        temp_file_path = os.path.join("/tmp", filename)
```

Comment:
I don't think that the `/tmp` directory exists on all operation systems.
Maybe we should use the [tempfile](https://docs.python.org/3/library/tempfile.html#module-tempfile) package instead? I guess, it would be more stable on various platforms.

Reply:
Alright, I'll make necessary changes.

Reply:
The method has been updated to use `tempfile`